src/content/changelogs/security-center.yaml

```
@@ -5,6 +5,9 @@ productLink: "/security-center/"
 productArea: Application security
 productAreaLink: /fundamentals/reference/changelog/security/
 entries:
+  - publish_date: "2025-01-20"
+    description: |-
+      - On the URL scanner, customers who search for a report can now see all reports related to a that specific hostname. A hash is also available in the security report. By selecting the hash, the dashboard will list reports containing the same hash.
```

```suggestion
      - On the URL scanner, customers who search for a report will now get a list of all reports related to that specific hostname. A hash is also available in the security report. By selecting the hash, the dashboard will list reports containing the same hash.
```
I'd avoid "can now see".